src/lib/ibcclient.ts

```diff
@@ -1368,13 +1382,18 @@ export interface CreateClientArgs {
   consensusState: TendermintConsensusState;
 }
 
+// this will query for the unbonding period.
+// if the trusting period is not set, it will use 2/3 of the unbonding period
 export async function buildCreateClientArgs(
   src: IbcClient,
-  unbondingPeriodSec: number,
-  trustPeriodSec: number
+  trustPeriodSec?: number
 ): Promise<CreateClientArgs> {
   const header = await src.latestHeader();
   const consensusState = buildConsensusState(header);
+  const unbondingPeriodSec = await src.getUnbondingPeriod();
+  if (!trustPeriodSec) {
+    trustPeriodSec = Math.floor((unbondingPeriodSec * 2) / 3);
+  }
```

Comment:
I think this is not a real case but if `trustPeriodSec` is explicitly set to `0` then it would result in 2/3 unbonding period. Should we check for undefined instead?

```suggestion
  if (trustPeriodSec === undefined) {
    trustPeriodSec = Math.floor((unbondingPeriodSec * 2) / 3);
  }
```

Reply:
I will make that clear. `trustPeriodSec` or 0 would make it unusable, but better to be clear with types.